src/UNL/Peoplefinder/Driver/LDAP/StandardFilter.php

```diff
@@ -48,9 +48,13 @@ function __construct($inquery, $operator = '&', $wild = false)
 
             //escape query
             $inquery = UNL_Peoplefinder_Driver_LDAP_Util::escape_filter_value($inquery);
-            
+
             //put the query into an array of words
             $query = preg_split('/\s+/', $inquery, 4);
+            if (count($query) > 1) {
+                //add the original multi-word query as a search term
+                $query[] = $inquery;
```

Comment:
The wild case is what I'm more worried about `('b bieber', '&' , true)`. 
I don't presume to know how LDAP handles queries, but I cannot logically see how the "NEW" filter didn't exclude all previous results, aside from LDAP magic.